src/components/Agent/AgentsList.vue

```diff
@@ -91,8 +91,7 @@ export default {
     }
   },
   methods: {
-    ...mapMutations('target', ['removeTargetEntityToDelete']),
-    ...mapMutations('general', ['addEntityToDelete']),
+    ...mapMutations('general', ['addEntityToDelete', 'removeTargetEntityToDelete']),

```

Comment:
why do we have on Agent component to use this target method removeTargetEntityToDelete ??